### PR TITLE
Rework Comet to export roots.json files for Webb3

### PR DESCRIPTION
**Pull request by scott-silver**
See https://github.com/compound-finance/webb3/pull/80

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,7 +26,6 @@
   },
   "scripts": {
     "build": "hardhat compile",
-    "prepare": "hardhat compile && tsc",
```
Comment (scott-silver) [inline package.json:26]: The `prepare` step gets called automatically when you install Comet as a dependency. However, the `hardhat compile` command requires all of the environment variables that we require in `hardhat.config.ts`.

As far as I can tell, we're not actually using this command anywhere. Removing it allows us to include Comet as a dependency in Webb3

File: package.json [hunk 2]
```diff
@@ -1,14 +1,17 @@
...
-    "dist"
+    "dist",
+    "deployments/**/roots.json"
   ],
   "exports": {
-    ".": "./dist/index.js",
-    "./*": "./dist/*.js"
+    ".":  "./dist/index.js",
+    "./*": "./dist/*.js",
+    "./kovan/roots.json": "./deployments/kovan/roots.json",
```
Comment (scott-silver) [inline package.json:13]: Annoyingly, you cannot specify a glob for a directory; this does not work:

```
"./**/roots.json": "./deployments/**/roots.json"
```

Comment (scott-silver) [reply package.json:13]: We might be able to reduce some of this duplication by exporting the entire `deployments/` directory, instead of each roots file

**Review by jflatow — APPROVED** [review]
Nice 👏

**Review by kevincheng96 — APPROVED** [review]

**Pull request merged**